src/deepsparse/v2/schedulers/scheduler.py

```diff
@@ -43,6 +43,8 @@ def submit(
         operator: Operator,
         operator_input: Any,
         context: Context,
+        pipeline_state: PipelineState,
+        inference_state: InferenceState,
```

Comment:
wonder if we should condense to `**kwargs` if possible to make plumbing future updates easier

Reply:
yah definitely going to be cleaning up args/types.